tempodb/encoding/vparquet/create.go

```diff
@@ -57,7 +57,7 @@ func CreateBlock(ctx context.Context, cfg *common.BlockConfig, meta *backend.Blo
 			return nil, err
 		}
 
-		if s.CurrentBufferLength() > cfg.RowGroupSizeBytes || s.CurrentBufferedObjects() > 10_000 {
+		if s.CurrentBufferedValues() > cfg.RowGroupSizeBytes {
```

Comment:
RowGroupSizeBytes...Ish :)

Reply:
Added a comment which helps a bit, and aligned it with the comment in compactor.go.